.github/workflows/build-pr.yml

```diff
@@ -155,24 +186,27 @@ jobs:
 
       - name: "Build, tag & push Angular frontend image"
         id: build-image
-        env:
-          COMPOSE_DOCKER_CLI_BUILD: 1
-          DOCKER_BUILDKIT: 1
-          PR_NUMBER: ${{steps.PR.outputs.number}}
-          REDIRECT_URL: "https://pr-${{steps.PR.outputs.number}}.pharmanetenrolment.gov.bc.ca"
+        uses: docker/build-push-action@v2
+        with:
+          context: prime-angular-frontend
+          push: true
+          file: prime-angular-frontend/openshift.dockerfile
+          tags: ${{secrets.OPENSHIFT_IMAGE_REPOSITORY}}/${{secrets.OPENSHIFT_LICENSE_PLATE}}-tools/prime-frontend:pr-${{steps.PR.outputs.number}}
+          secrets:
```

Comment:
Aren't all of these available as secrets?

Reply:
See below for a real example.

Reply:
not quite. these are just build arguments. so should be `build-args` really. I'll correct this.